subseas_flex_fcst/config-zmd.yaml

```diff
@@ -0,0 +1,24 @@
+# App set up
+server: 0.0.0.0
+port: 8063
```

Comment:
Why does this one specify a port and the others don't?

Reply:
no reason. I wasn't paying attention to changed to the country specific config files